src/doc/using_visit/Operators/OperatorTypes/ExtrudeStacked_operator.rst

```diff
@@ -0,0 +1,106 @@
+.. _ExtrudeStacked operator:
+
+ExtrudeStacked operator
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The ExtrudeStacked operator uses one or more scalar fields on a 2D
```

Comment:
We have a policy of a single sentence per line. Can you update accordingly, please.

Reply:
I do not understand this requirement. Would you provide an example.

Reply:
The following, which has two sentences spread over four lines:
```
The ExtrudeStacked operator can be used to create 3D height field
plots by applying the operator to a plot that accepts scalar
values. The ExtrudeStacked operator is quite flexible because it can
extrude by more than one variable and displayed different ways.
```

should be as below, where each sentence is on one line, regardless of its length:
```
The ExtrudeStacked operator can be used to create 3D height field plots by applying the operator to a plot that accepts scalar values. 
The ExtrudeStacked operator is quite flexible because it can extrude by more than one variable and displayed different ways.
```

This requirement is documented in our [Contributing To Documentation](https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/ContributingToDocumentation/index.html) with an expanded explanation related to [line-length](https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/ContributingToDocumentation/index.html#contributing-line-length).

Sphinx will take care of the wrapping as needed.

Reply:
Thanks I understand now but in looking at some of the other operators and plots none of them have that formatting.

Reply:
Yes, the policy was introduced after we had already created some of the content. It was decided that those files would be made to comply with the policy either when someone updated the file, or during a future code-sprint.
New files must comply.

Reply:
Note to self: Do not ever touch a rst file again :-)